iogt/views.py

```diff
@@ -109,6 +110,15 @@ def get(self, request, page_id):
         page = get_object_or_404(Page, id=page_id)
         pages = page.get_descendants(inclusive=True).live().specific()
         page_urls = []
+        active_locale = Locale.get_active()
```

Comment:
I am wondering why there are no corresponding changes to the tests.

Reply:
what I mean here is that there are additional URLs that must be included, but there is no test to make sure that they are. Is there a good reason for not changing the tests?

Reply:
I have added/updated tests cases to incorporate these cases too